### DISCUSSION TITLE
Implementing New Currency - BitcoinZ

### DISCUSSION THREAD

**Discussion by bucketphobia** [coinfoundry/miningcore#858] [Ideas]
Hello, I was wondering if it was possible to add BitcoinZ support to the miner.

https://github.com/bitcoinz-pod/bitcoinz

Thanks!

**Comment by martin-key** [comment]
What are the differences with Zcash ?

**Comment by martin-key** [comment]
From what I see this is rebranded copy of Zcash. However they didn't leave the original commits, so changes are really hard to be observed.

**Comment by Gadflyii** [comment]
@oliverw @StarbuckBG 

What is the best way to add a new coin? Is there a coin template or coin settings I am missing somewhere?

**Comment by calvintam236** [comment]
@Gadflyii https://github.com/coinfoundry/miningcore/issues/85

**Comment by michaelotis** [comment]
@StarbuckBG The one change is a block reward of 12500 coins due to the 21 bln supply
